modules/ROOT/pages/graph-analytics/index.adoc

```diff
@@ -51,19 +68,42 @@ Security patches and new versions of GDS and Neo4j are installed within short ti
 The operations are non-destructive, so graph projections, models, and data present on an instance are not affected.
 No operation is applied until all the running GDS algorithms have completed.
 
-== Comparison
+[[aura-gds-no-db]]
+== {gds-no-db-name}
 
-There are important operational differences between the Graph Analytics plugin on AuraDB Professional and AuraDS.
+{gds-no-db-name} allows you to use the link:https://neo4j.com/docs/graph-data-science/current/introduction/[Graph Data Science library] on Aura without AuraDB or AuraDS, on any supported cloud providers and regions.
 
-[opts="header", cols="1,1"]
-|===
-|AuraDB Pro + Graph Analytics plugin
-|AuraDS
+Graph Analytics TBD will run on one or more instances, optimised for analytics workloads.
+You can import your data [...]
+You can control the shape and the number of these instances, and will only be billed for what you consume.
 
-|Rolling updates clear all projections.
-|Rolling updates save projections to disk and restore them upon restart.
+== Comparison
 
-|Backup files do not include projections.
-|Backup files include all projections.
+[opts="header", cols="1,1,1,1"]
+|===
+|Graph Analytics plugin
+|Graph Analytics Serverless
+|AuraDS
+|{gds-no-db-name}
```

Comment:
I don't understand why this is its own option. I suggest we remove it. There's no technical difference in the service experience of the `Serverless` which depends on the data source used, except `write` will be off-limits.
```suggestion
```